railties/lib/rails/application/default_middleware_stack.rb

```diff
@@ -16,6 +16,11 @@ def build_stack
           middleware.use ::ActionDispatch::HostAuthorization, config.hosts, config.action_dispatch.hosts_response_app
 
           if config.force_ssl
+            if config.action_dispatch.ssl_redirect_status_308 && !config.ssl_options.dig(:redirect, :status)
+              config.ssl_options[:redirect] ||= {}
+              config.ssl_options[:redirect][:status] = 308
```

Comment:
@pixeltrix I didn't make any changes to `ssl.rb` because I'm overriding the status here.